corehq/apps/data_dictionary/static/data_dictionary/js/data_dictionary.js

```diff
@@ -606,6 +613,33 @@ var dataDictionaryModel = function (dataUrl, casePropertyUrl, typeChoices, fhirR
         }
     };
 
+    self.showCasePropertyWarningIfNeeded = function () {
+        let caseType = self.getActiveCaseType();
+        if (caseType.propertyCount > self.casePropertyLimit) {
+            let content = gettext(
+                "The '" + caseType.name + "' case has " + caseType.propertyCount + " properties. " +
+                "We recommend keeping case properties below " + self.casePropertyLimit + ", " +
```

Comment:
Translating a string containing pre-substituted variable values will not work, right?

Reply:
[512de98](https://github.com/dimagi/commcare-hq/pull/36588/commits/512de98a6a4c9cf34ed503afc20941d5dbd81937)